cmd/virt-launcher/virt-launcher.go

```diff
@@ -335,6 +337,21 @@ func waitForFinalNotify(deleteNotificationSent chan watch.Event,
 	}
 }
 
+func runDownwardMetricsVirtioServer(ctx context.Context) error {
+	if _, ok := os.LookupEnv("VIRT_LAUNCHER_DOWNWARDMETRICS_SERVER"); ok {
```

Comment:
make a constant from "VIRT_LAUNCHER_DOWNWARDMETRICS_SERVER"?

Reply:
Thanks for your review! 

Actually, we have created constants in pkg/virt-controller/services/template.go for those envars. I have changed those strings for constants `ENV_VAR_VIRT_LAUNCHER_DOWNWARDMETRICS_SERVER` and `ENV_VAR_NODE_NAME`. Does it look good to you?